plugins/PostProcessingPlugin/scripts/ColorMix.py

```diff
@@ -0,0 +1,190 @@
+# ColorMix script - 2-1 extruder color mix and blending
+# This script is specific for the Geeetech A10M dual extruder but should work with other Marlin printers. 
+# It runs with the PostProcessingPlugin which is released under the terms of the AGPLv3 or higher.
+# This script is licensed under the Creative Commons - Attribution - Share Alike (CC BY-SA) terms
```

Comment:
Since your script is based off code that is licenced under GPL, yours can not be licenced under CC-BY-SA (as it removes some restrictions from GPL). We unfortunately can't merge with that statement in.

Reply:
This script is under CC-BY-SA, please refer to ChangeAtZ.py in the script folder for reference.  The plugin that runs these script's is GPL.
Thanks

Reply:
Huh. That's weird. Those licences bite oneother.

Reply:
We decided to leave it as is for now, as you correctly point out; it's already in ChangeAtZ.